client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,24 @@
 import React from 'react';
+const MovieCard = ({ movie }) => {
+  const { title, director, metascore, stars, id } = movie;
```

Comment:
nice job refactoring this into its own component and destructuring the movie from props, the the title director metascore stars and id from that object so you didnt have to drill into it like props.movie.title